src/screens/ProfileScreen/index.js

```diff
@@ -640,7 +641,6 @@ const ProfileScreen = ({route}) => {
   };
 
   const onDeletePost = async () => {
-    setIsOptionModalOpen(false);
     removePostByIdFromContext();
 
     let response;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `setIsOptionModalOpen(false);` has been removed from the `onDeletePost` function. If this state update is necessary for the correct functioning of the modal, it should be reintroduced.
```diff
- setIsOptionModalOpen(false);
+ removePostByIdFromContext();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->